encoding.bs

```diff
@@ -28,6 +28,18 @@ Translate IDs: dictdef-textdecoderoptions textdecoderoptions,dictdef-textdecodeo
 spec:infra; type:dfn;
     text:code point
     text:ascii case-insensitive
+spec:streams;
+    type:interface; text:ReadableStream
+    type:dfn; text:chunk
+    type:dfn; text:readable stream
+    type:dfn; text:writable stream
```

Comment:
Why are these needed? Do you get some conflict that cannot be resolved?

Reply:
"ReadableStream" is also exported by Fetch, and so needs to be disambiguated. I have modified the Streams Standard to export "chunk", "readable stream" and "writable stream" in https://github.com/whatwg/streams/pull/871, so those won't be needed once Bikeshed has picked up the change.

Reply:
Shall we make Fetch stop doing that then? It doesn't really seem like Fetch's way of managing it handles anything.

Reply:
@yutakahirano Who is using the ReadableStream definition that is exported from Fetch?

Reply:
I find none, but serviceworker uses related ones ("construct a ReadableStream object", "empty ReadableStream"). See also: https://github.com/heycam/webidl/issues/445